handlebars/pom.xml

```diff
@@ -89,6 +103,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
+      <scope>provided</scope>
```

Comment:
Sure, makes it a fair bit easier, I was just trying to do the least
disruptive thing to the code base. I will close this PR and make a new one
with a different POM.

On Tue, Oct 6, 2015 at 2:12 PM Edgar Espina <notifications@github.com>
wrote:

> In handlebars/pom.xml
> <https://github.com/jknack/handlebars.java/pull/409#discussion_r41300889>:
>
> > @@ -89,6 +103,7 @@
> >      <dependency>
> >        <groupId>org.apache.commons</groupId>
> >        <artifactId>commons-lang3</artifactId>
> > +      <scope>provided</scope>
>
> of course, something like: handlebars-osgi? can you please do it?
>
> thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jknack/handlebars.java/pull/409/files#r41300889>.
>